templates/geocontext/0/rancher-compose.yml

```diff
@@ -0,0 +1,117 @@
+version: '2'
+catalog:
+  name: "Kartoza BIMS"
+  description: "Kartoza's Biodiversity Information Management System."
```

Comment:
And this to "Kartoza's Geocontext App - discover hierarchical information for any place on your map."